test/rubocop.t

```diff
@@ -9,16 +9,21 @@ rubocop
   
   Offenses:
   
+  four.rb:1:1: C: [Corrected] Style/FrozenStringLiteralComment: Missing magic comment # frozen_string_literal: true.
+  # bad - four spaces
+  ^
```

Comment:
The changes in this file make me nervous. Is a version not pinned somewhere that ought to be?

Reply:
I believe so, yes: https://github.com/restyled-io/restylers/blob/dd9fdfaf8fd10e76a10c8cfac40c038160f8c919/rubocop/Dockerfile#L4

Reply:
Ah boo. OK, I think I'll pin it and update the test expectations if needed separately.